src/user.jl

```diff
@@ -20,15 +20,32 @@ function encode_attachment(filename::String)
         content_disposition = "attachment"
     end
 
-    encoded_str = 
+    # Some email clients, like Spark Mail, have problems when the attachment
+    # encoded string is very long. This code breaks the payload into lines with
+    # 75 characters, avoiding those problems.
+    raw_attachment = String(take!(io))
+    buf = IOBuffer()
+    char_count = 0
+
+    for c in raw_attachment
```

Comment:
can't this loop be written on the buffer (here `io` ) . We have a buffer which we convert to a string, then write back to a buffer and then convert to string once again. Which seems a bit redundant.

Reply:
Hi @aviks ! Probably not. The first buffer encodes the file. Hence, every byte written to it is automatically encoded, meaning we cannot add a break-line to the encoded stream. This functionality should be implemented in `Base64` module of Julia. At the second stage, to break the long lines, we could use just a string and concatenate every character using `*`. However, a buffer for this case is much faster.

Reply:
Hi @aviks !

Can we merge as it is or do you have a new proposal?

Reply:
I had some thoughts, but clearly I haven't had the time to experiment with that 😄